orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -193,3 +193,15 @@ def add_module(self, name: str, module: Optional['Module']) -> None:
         """Raises a NotImplementedError exception since ORTModule does not support adding modules to it"""
 
         raise NotImplementedError("ORTModule does not support adding modules to it.")
+
+    @property
+    def module(self):
+        """ORTModule attribute to access the original module
+
+        HuggingFace Trainer `save_model` method checks to see if the input model is a HuggingFace PreTrainedModel
+        or if the model has an attribute called `module` which references a HuggingFace PreTrainedModel to save
+        the entire context of the model so that it can be loaded using HuggingFace `from_pretrained` method.
+        This `module` property enables HuggingFace Trainer to retrieve the underlying PreTrainedModel inside ORTModule
+        to save and load a complete checkpoint"""
+
```

Comment:
I'd suggest trimming this comment down, since a casual reader of the docs doesn't need to know the rationale for this PR. 

```suggestion
    def module(self) -> torch.nn.Module:
        """The original `torch.nn.Module` that this module wraps.
        
        This property provides access to methods and properties on the original module.
        """
```

Reply:
Maybe downgrade to a `# internal comment`?